cli/integration-test/integration/definition_test.clj

```diff
@@ -105,3 +105,26 @@
         (-> (lsp/request! (fixture/hover-external-uri-request uri 7612 5))
             :contents
             (get 1))))))
+
+(deftest definition-external-dependency-zipfile-scheme-escaped-uris
+  (h/delete-project-file "../../.lsp/.cache/")
+  (h/set-escape-uris! true)
```

Comment:
I thought about passing a param but would need lots of changes saying that the URI should be encoded, so a dynamic var was less worst 😅

Reply:
Would `with-redefs` work? That'd let you avoid having to remember to set the var back to `false` at the end of the test.

Reply:
I think so, for a moment I thought it was not thread-safe